app/src/main/java/com/hover/stax/faq/FAQAdapter.kt

```diff
@@ -0,0 +1,46 @@
+package com.hover.stax.faq
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.hover.stax.databinding.FaqItemsBinding
+
+class FAQAdapter(private val faqs: List<FAQ>, private val selectListener: SelectListener):
+        RecyclerView.Adapter<FAQAdapter.Companion.FAQViewHolder>() {
+
+    companion object {
```

Comment:
The viewholder can be defined as an inner class to allow for access to it and the listener. Then you don't have to wrap it in a companion object.